server/utils/db/connection_pool.py

```
@@ -169,7 +170,49 @@ def get_connection(self):
             if connection:
                 try:
                     connection.rollback()
-                    with connection.cursor() as cur:  # No RLS needed — pool cleanup (RESET vars)
+                    with connection.cursor() as cur:
+                        cur.execute(
+                            "RESET ROLE; RESET myapp.current_user_id; RESET myapp.current_org_id;"
+                        )
+                    connection.commit()
+                except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)